### ISSUE TITLE
Show who the LPs are and what their share of the pool is

### ISSUE THREAD

**Issue by maltekliemann** [zeitgeistpm/ui#1536]
I'd like the following info:

- Important: What's _my_ share of the LP and what's the total?
- Useful/somewhat vain: Who are the other LPs and what are their shares.

The occasion is that I've withdrawn all my shares from a live market and actually wasn't sure how much of the pool I'm withdrawing, and I'm also wonder who the other LPs are. A pie chart would be awesome.

**Comment by Robiquet** [comment]
Why is it important to know what percentage of the pool you own?

**Comment by maltekliemann** [comment]
Hmm, maybe "important" was a bit too much. You need to know what your share is to get a guess of how much of the trading fees you'll receive. I think it's also just interesting to see how much you're involved in this market. I feel like it's similar to knowing the trading volume as a trader. For example, if I just bought YES for 1,000 ZTG, I might want to know if that was a large buy, or a medium buy, etc. I guess as LP you can figure this out by looking at the total liquidity, but I think it would be nice to just have an overview.

Considering this as low priority though.

**Comment by Robiquet** [comment]
Yeah I think something like a "Biggest Participants" section would be cool. It could show the biggest token and LP holders
